src/rpc/client.cpp

```diff
@@ -122,6 +122,7 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "createrawtransaction", 1, "outputs" },
     { "createrawtransaction", 2, "locktime" },
     { "createrawtransaction", 3, "replaceable" },
+    { "createrawtransaction", 4, "version" },
```

Comment:
Probably best not to have this be a positional parameter.

Reply:
No, the issue isn't this one line of code, but the overall addition of the parameter

Reply:
I agree think it's a bit too important for a 4th positional arg

Reply:
@luke-jr @adamandrews1 
So should I remove the line? If I should, could you advise which code should be updated to avoid failure of the aforementioned validation?

Reply:
Related, see https://github.com/bitcoin/bitcoin/pull/31936#discussion_r2010279314

Reply:
This is needed as the arg is non-string type. My understanding is that @luke-jr is concept nacking the PR. I don't really understand the comments about the positionalness of this arg?